bzt/engine.py

```diff
@@ -1063,6 +1066,17 @@ def __init__(self):
         self.start_time = None
         self.preprocess_args = lambda x: None
 
+        self.features = {
+            self.TRANSACTION_ABLE: False
+        }
+
+    def get_requests(self, parser=RequestParser, scenario=None):
```

Comment:
Moving get_requests into executor level looks like change architectural concepts... Does it have enough reasons?